wagn/lib/wagn/generators/wagn/wagn_generator.rb

```diff
@@ -79,17 +77,6 @@ def rakefile
     template "Rakefile"
   end
 
-  def app
-    empty_directory 'app'
-    inside "app" do
-      empty_directory 'assets'
-      inside "assets" do
-        empty_directory 'javascripts'  
-        empty_directory 'stylesheets'
-      end
-    end
-  end
-
```

Comment:
Exoneration!  That actually makes more sense.  (Not to have that in the decks, but it would have gone in there.)  Bad gatekeeping on my part :(